forge-gui/res/cardsfolder/upcoming/scheming_fence.txt

```diff
@@ -0,0 +1,9 @@
+Name:Scheming Fence
+ManaCost:W U
+Types:Creature Human Citizen
+PT:2/3
+K:ETBReplacement:Other:ChooseCard
+SVar:ChooseCard:DB$ ChooseCard | Choices$ Permanent.nonLand | MinAmount$ 0 | ChoiceTitle$ You may choose a nonland permanent | AILogic$ OppPreferred | SpellDescription$ As CARDNAME enters the battlefield, you may choose a nonland permanent.
+S:Mode$ Continuous | Affected$ Card.ChosenCard | AddHiddenKeyword$ CARDNAME's activated abilities can't be activated. | Description$ Activated abilities of the chosen permanent can't be activated.
+S:Mode$ Continuous | Affected$ Card.Self | EffectZone$ Battlefield | GainsAbilitiesOfDefined$ ChosenCard | GainsAbilitiesActivateIgnoreColor$ True | Description$ CARDNAME has all activated abilities of the chosen permanent except for loyalty abilities. You may spend mana as though it were mana of any color to activate those abilities.
```

Comment:
`Except Loyatly Abilities`, you probably need a isValid filter in the Class

Reply:
I thought this boolean would take care of that, but apparently not :(
![image](https://user-images.githubusercontent.com/103371817/165340461-ea083f8d-aaf6-4dee-af85-6c3899e503ee.png)

Reply:
yeah, better replace it with a Valid Check that can be better extended